packages/grafana-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,61 +1,189 @@
-import React, { createRef, FC } from 'react';
-import { VirtualElement } from '@popperjs/core';
-import { Popover } from './Popover';
-import { PopoverController, UsingPopperProps } from './PopoverController';
-import { closePopover } from '../../utils/closePopover';
+import React from 'react';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { useStyles2 } from '../../themes/ThemeContext';
+import { PopoverContent, TooltipPlacement } from './types';
+import { Portal } from '../Portal/Portal';
 
-export interface TooltipProps extends UsingPopperProps {
+export interface TooltipProps {
   theme?: 'info' | 'error' | 'info-alt';
+  show?: boolean;
+  placement?: TooltipPlacement;
+  content: PopoverContent;
+  children: JSX.Element;
+  /**
+   * Set to true if you want the tooltip to stay long enough so the user can move mouse over content to select text or click a link
+   */
+  interactive?: boolean;
 }
 
-export interface PopoverContentProps {
-  updatePopperPosition?: () => void;
-}
-
-export type PopoverContent = string | React.ReactElement<any> | ((props: PopoverContentProps) => JSX.Element);
+export const Tooltip = React.memo(({ children, theme, interactive, show, placement, content }: TooltipProps) => {
+  const { getArrowProps, getTooltipProps, setTooltipRef, setTriggerRef, visible, update } = usePopperTooltip({
+    visible: show,
+    placement: placement,
+    interactive: interactive,
+    delayHide: interactive ? 100 : 0,
+    delayShow: 50,
+    // Focus is new, old tooltip did not show on focus, we could make this a parameter/option?
+    trigger: ['hover', 'focus'],
```

Comment:
From the a11y POV I think we should keep the focus to open the tooltips automatically. Can't think of any situation when provided tooltip would not be shown in keyboard navigation scenario.

So IMO the behavior presented by this PR is desired one.